doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -197,6 +210,34 @@ with a ``.ipynb`` file.
 
 .. _MyST-NB: https://myst-nb.readthedocs.io/
 
+.. _interactive-docs:
+
+Interactive examples in docstrings
+----------------------------------
+
+Examples within docstrings can be made interactive using the [``jupyterlite-sphinx`` extension](https://jupyterlite-sphinx.readthedocs.io/en/latest/).
```

Comment:
This isn't rendering.
![image](https://github.com/user-attachments/assets/efcea123-b23f-4f16-a8a0-2e386138212a)

Reply:
OOps, my bad.